src/bilder/images/edxapp/templates/edxapp/xpro/common_values.yml

```diff
@@ -0,0 +1,577 @@
+                default_class: xmodule.hidden_module.HiddenDescriptor
+                fs_root: /edx/var/edxapp/data
+                render_template: common.djangoapps.edxmako.shortcuts.render_to_string
+            - ENGINE: xmodule.modulestore.mongo.DraftMongoModuleStore
+              NAME: draft
+              DOC_STORE_CONFIG:
+                <<: *docstore_config
+              OPTIONS:
+                default_class: xmodule.hidden_module.HiddenDescriptor
+                fs_root: /edx/var/edxapp/data
+                render_template: common.djangoapps.edxmako.shortcuts.render_to_string
+ORA2_FILE_PREFIX: xpro/ora2  # MODIFIED
+PARTNER_SUPPORT_EMAIL: ''
+PASSWORD_POLICY_COMPLIANCE_ROLLOUT_CONFIG:
+    ENFORCE_COMPLIANCE_ON_LOGIN: false
+PASSWORD_RESET_SUPPORT_LINK: ''
+PAYMENT_SUPPORT_EMAIL: {{ key "edxapp/sender-email-address" }}  # MODIFIED
+PLATFORM_DESCRIPTION: MIT xPRO # MODIFIED
+PLATFORM_FACEBOOK_ACCOUNT: http://www.facebook.com/YourPlatformFacebookAccount
+PLATFORM_NAME: MIT xPRO  # MODIFIED
+PLATFORM_TWITTER_ACCOUNT: '@YourPlatformTwitterAccount'
+POLICY_CHANGE_GRADES_ROUTING_KEY: edx.lms.core.default
+PRESS_EMAIL: support@xpro.mit.edu # MODIFIED
+PROCTORING_BACKENDS:
+    DEFAULT: 'proctortrack'
+    'proctortrack':
+      {{ with secret "secret-xpro/edxapp" }}
+      'client_id': '{{ .Data.proctortrack_client_id }}'
+      'client_secret': '{{ .Data.proctortrack_client_secret }}'
+      {{ end }}
+      'base_url': '{{ key "edxapp/proctortrack-base-url" }}'
+    'null': {}
+
+PROCTORING_SETTINGS: {}
+REGISTRATION_EXTRA_FIELDS:
+    city: hidden
+    confirm_email: hidden
+    country: hidden
+    gender: optional
+    goals: optional
+    honor_code: hidden
+    level_of_education: optional
+    mailing_address: hidden
+    terms_of_service: hidden
+    year_of_birth: optional
+RETIRED_EMAIL_DOMAIN: retired.invalid
+RETIRED_EMAIL_PREFIX: retired__user_
+RETIRED_USERNAME_PREFIX: retired__user_
+RETIRED_USER_SALTS:
+- OVERRIDE ME WITH A RANDOM VALUE
+- ROTATE SALTS BY APPENDING NEW VALUES
+RETIREMENT_SERVICE_WORKER_USERNAME: retirement_worker
+RETIREMENT_STATES:
+- PENDING
+- ERRORED
+- ABORTED
+- COMPLETE
+SEGMENT_KEY: null
+SENTRY_ENVIRONMENT: {{ env "ENVIRONMENT" }}
+SERVER_EMAIL: odl-devops@mit.edu # MODIFIED
+# Removing the session cookie domain as it is no longer needed for sharing the cookie
+# between LMS and Studio (TMM 2021-10-22)
+# SESSION_COOKIE_DOMAIN: {{ key "edxapp/cookie-domain" }} # MODIFIED
+SESSION_COOKIE_SECURE: true # MODIFIED
+SESSION_SAVE_EVERY_REQUEST: false
+SITE_NAME: MIT xPRO  # MODIFIED
+SOCIAL_MEDIA_FOOTER_URLS: {}
+SOCIAL_SHARING_SETTINGS:
+    CERTIFICATE_FACEBOOK: false
+    CERTIFICATE_TWITTER: false
+    CUSTOM_COURSE_URLS: false
+    DASHBOARD_FACEBOOK: false
+    DASHBOARD_TWITTER: false
+STATIC_ROOT_BASE: /edx/var/edxapp/staticfiles
+STATIC_URL_BASE: /static/
+STUDIO_NAME: Studio
+STUDIO_SHORT_NAME: Studio
+SUPPORT_SITE_LINK: https://mitx-micromasters.zendesk.com/hc/en-us/requests/new
```

Comment:
?

Reply:
should be https://xpro.zendesk.com/hc I believe